.github/workflows/build-and-test-multiplatform.yml

```
@@ -0,0 +1,796 @@
+      uses: actions/upload-artifact@v4.4.3
+      with:
+        name: install-macos-gcc-x64
+        path: install
+
+  prepare-and-build-linux-clang-x64:
+    name: Prepare and Build on Ubuntu with Clang (x64)
+    runs-on: ubuntu-24.04
+
+    steps:
+    # Step: Checkout the repository
+    - name: Checkout repository
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)